.php_cs.dist

```diff
@@ -11,8 +11,6 @@ return PhpCsFixer\Config::create()
         '@Symfony' => true,
         '@Symfony:risky' => true,
         'protected_to_private' => false,
-        'native_constant_invocation' => true,
```

Comment:
in continuation of https://github.com/symfony/symfony/pull/39772/files#r554500252 ,

@nicolas-grekas

https://cs.symfony.com/doc/rules/function_notation/native_function_invocation.html#strict